### ISSUE TITLE
Page moves when a drop down search item is selected

### ISSUE THREAD

**Issue by aeggum** [mswysocki/Madison-Student-Apartment-Finder#8]
This only happens the first time a drop-down is opened up.  
All others (after the first) look like they are just fine.

**Comment by xibaca** [comment]
Which browser are you using?
I could not see this behavior at msaf.heroku.com with Chrome.

**Comment by aeggum** [comment]
I get the issue no matter what field, on Firefox (12) on Ubuntu.  Chrome on the same machine I get the error to happen from the third drop-down option on down (ie. 'Type of Residence') and below..weird. I'll check on my Mac and see what kind of results I get the issue on Chrome on the mac but not Firefox.  Someday if I'm inclined I'll check it out on Safari.

**Comment by xibaca** [comment]
I think I now what you mean.
And I know why it is not happening here at my browser.

I may be wrong, but is the resolution in your computer high, so that when
you browse to msaf.heroku.com, there is no scrolling side bar in the
right? Then, when you click on some options they show up and the page gets
bigger so that the scrolling appears. And the page is pushed a little bit
to the left.

In here, I saw that happening before (I think at the Poseidon machines) but
here at my computer, when the page is loaded, the scrolling bar in the left
is already there.

I think we can control it via javascript, maybe forcing the bar to be there
always or disabling it and designing our own scroll bar, or doing something
else (even without javascript).

2012/3/24 Adam Eggum <
reply@reply.github.com

> I get the issue no matter what field, on Firefox (12) on Ubuntu.  Chrome
> on the same machine I get the error to happen from the third drop-down
> option on down (ie. 'Type of Residence') and below..weird. I'll check on my
> Mac and see what kind of results I get the issue on Chrome on the mac but
> not Firefox.  Someday if I'm inclined I'll check it out on Safari.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/mswysocki/Madison-Student-Apartment-Finder/issues/8#issuecomment-4674700

## 

Guilherme Santos G. Baptista
Graduando em Cincia da Computao pela UFCG
LSD - Laboratrio de Sistemas Distribudos

**Comment by aeggum** [comment]
I just have a 1440x900 resolution screen, but yes, that's what happens.. when the scroll bar shows up the page jumps..good catch! I missed that and just noticed the jumping.

Sounds good!

**Comment by xibaca** [comment]
Nice!, I just closed the issue then.

**Comment by aeggum** [comment]
Was this issue resolved or you know how to fix it or something?
Just wondering as I still get the slight page move

**Comment by xibaca** [comment]
No, I just closed because I thought we would leave it that way, but we'd better solve this. The page move is not something beautiful... We will have to see how to do that.

**Comment by aeggum** [comment]
I don't think this is really an issue any more.  It might still happen but it happens very slowly and isn't as obnoxious.
